### DISCUSSION TITLE
Re-Order Option (Reorder for Ordered Items)

### DISCUSSION THREAD

**Discussion by devomman** [inspireui/support#5334] [New Feature/Ideas Requests]
Hi, Please add for re-order if payment success or not success or pending or cancelled status of order reorder same order details ex. plugin: https://wordpress.org/plugins/one-click-order-reorder/

Thanks

**Comment by genuinesourcecode** [comment]
This is Great Features for Reorder Option Thanks for Suggestions, We also Agreed

**Comment by krrisharora** [comment]
agreed
